src/players/VAST.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import videojs from 'maestro-videojs-vast';
+import 'maestro-videojs-vast/dist/maestroVideoJsVast.css';
+import { callPlayer } from '../utils';
+import createSinglePlayer from '../singlePlayer';
+
+const PLACEHOLDER_VIDEO = 'https://storage.googleapis.com/maestro_video/blank.mp4';
```

Comment:
we got a way we have been handling binaries for the App... wonder if it makes sense to use that

Reply:
i don't know how to do that so i'll need some guidance on adding this